### ISSUE TITLE
yield_to fails when code emits it with an argument

### ISSUE THREAD

**Issue by nightblade9** [bitwes/Gut#327]
**TL;DR** calling `yield(yield_to(some_thing, "some_signal", 1.0), YIELD)` fails and errors out if `some_thing` emits `some_signal` with a value (e.g. `emit_signal("some_signal", "hi there!")`).

Longer version:

I have several tests that started failing with errors similar to:

`ERROR: Error calling method from signal 'moved': 'Control(gut.gd)::_yielding_callback': Method expected 1 arguments, but called with 2..`

The simplest way to understand what's happening is that my "production code" looks like this:

```gdscript
signal moved(new_position)

func do_stuff():
  yield("moved", Vector2(37, 10))
```

In my test code, I simply use:

```gdscript
func test_stuff():
  var thing = Thing.new()
  thing.do_stuff()
  yield(yield_to(thing, "moved", 1.0), YIELD)
```

This fails with the error indicated above. I'm 99% sure it's because I started emitting `moved` with some parameter. There doesn't seem to be an easy way around this. Any help would be appreciated...

**Comment by bitwes** [comment]
This was fixed in #258 and released in 7.2.0 (October 25, 2021).

**Comment by nightblade9** [comment]
My bad. I even commented on that issue. Upgrading resolved my issue, thank you!

**Comment by bitwes** [comment]
No problem.  I forgot when I did this and had to look up the git-blame to see when it was released.  Then I re-found 258.

**Comment by jamie-pate** [comment]
This still seems broken to me?

Connects a signal to a method on a target object. Pass optional binds to the call as an [Array](https://docs.godotengine.org/en/3.3/classes/class_array.html#class-array) of parameters. These parameters will be passed to the method **after any parameter used in the call** to [emit_signal](https://docs.godotengine.org/en/3.3/classes/class_object.html?highlight=object#class-object-method-emit-signal). Use flags to set deferred or one-shot connections. See [ConnectFlags](https://docs.godotengine.org/en/3.3/classes/class_object.html?highlight=object#enum-object-connectflags) constants.

https://github.com/bitwes/Gut/blob/v7.4.1/addons/gut/gut.gd#L323 <- from_obj is the **first** argument in the callback
